Tests/scripts/add_secret_file_to_build.py

```diff
@@ -0,0 +1,61 @@
+import argparse
+import json
+
+from Tests.scripts.utils.GoogleSecretManagerModule import GoogleSecreteManagerModule
+
+
+def run(options):
+    json_path_file = options.service_account
+    print(f'******************conf file location: {json_path_file}')
```

Comment:
Also, remove all the *'s before merging.
```suggestion
    print(f'****************** conf file location: {json_path_file}')
```